buildly/settings/authentication.py

```diff
@@ -81,11 +78,13 @@
 SOCIAL_AUTH_URL_NAMESPACE = 'social'
 SOCIAL_AUTH_POSTGRES_JSONFIELD = True
 
-SOCIAL_AUTH_REDIRECT_IS_HTTPS = True if os.getenv('SOCIAL_AUTH_REDIRECT_IS_HTTPS') == 'True' else False
+SOCIAL_AUTH_REDIRECT_IS_HTTPS = (
```

Comment:
so many of these line break changes I want to fire myself and make whoever did these work on all the conflicts.  Let's write a single style guide for pythonic changes and only make them during refactor releases.